packages/amazonq/package.json

```diff
@@ -1,37 +1,21 @@
 {
     "name": "amazon-q-vscode",
-    "displayName": "Amazon Q (Preview) + CodeWhisperer",
-    "description": "Amazon Q (Preview) + CodeWhisperer",
-    "version": "1.0.0-SNAPSHOT",
+    "displayName": "Standalone Test",
+    "description": "",
+    "version": "0.1.0-SNAPSHOT",
```

Comment:
Good idea. I noticed that MS marketplace doesn't allow you to re-publish an existing version, so if we were to publish 1.0.0 we could not overwrite it.